src/dotnetTemplate/Microsoft.PowerShell.PowerShellStandard.Template/Microsoft.PowerShell.PowerShellStandard.Template/InvokeNoun.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+
+namespace Microsoft.PowerShell.PowerShellStandard.Template
+{
+    [Cmdlet("Invoke","Noun")]
+    public class InvokeNoun : PSCmdlet
```

Comment:
At least with PowerShell, convention is to suffix with Command potentially making it easier to find in complex code.  So `TestSampleCmdletCommand`